test/CarouselSpec.js

```diff
@@ -490,6 +562,19 @@ describe('<Carousel>', () => {
         </Carousel>,
       );
 
+      wrapperWithCustomEvents = mount(
+        <Carousel
+          activeIndex={1}
+          interval={null}
+          onTouchStart={onSelectSpy}
+          onTouchMove={onSelectSpy}
+          onTouchEnd={onSelectSpy}
+          touch
+        >
+          {items}
+        </Carousel>,
+      );
```

Comment:
Can we use the existing wrapper?  Also, best to not reuse the same spy for all the different events.